app/lib/log_util.rb

```diff
@@ -1,10 +1,16 @@
+# frozen_string_literal: true
+
 class LogUtil
-  def self.log_err_and_airbrake(msg)
-    Rails.logger.error(msg)
+  def self.log_err_and_airbrake(msg, exception = nil)
+    LogUtil.log_backtrace(exception, msg)
     Airbrake.notify(msg)
```

Comment:
I thought airbrake would automatically send the trace, no?

Reply:
No. These calls might eventually be "injecting" fragments of the exception on that `msg` string, or eventually printing the stack trace in a separate call, which make it very hard to correlate these "airbrake error messages" to their actual stack traces (and there are places where the backtrace and exception message are being completely masked)